proto/v1/setting_service.proto

```diff
@@ -293,3 +294,19 @@ message DataClassificationSetting {
   }
   repeated DataClassificationConfig configs = 1;
 }
+
+message DataCategorySetting {
+  message DataCategoryItem {
+    // id is the uuid for category item.
+    string id = 1;
```

Comment:
Can we use the title as the unique identifier of `DataCategoryItem `?

Reply:
I'm not sure if the `title` is the classification title. Looks like in the data feature tab, we need to select the classification and set the data masking algorithm.
If so, the title should be the classification title, and seems we need the uuid for `DataCategoryItem`

Reply:
> I'm not sure if the title is the classification title.

It is the title of the data category, for example, `Address`, `Phone Number`. I think the data category is the subcategory of the data classification, I can say the two data category items above are all belong to `Communication` data classification.